app/src/main/java/com/nervousfish/nervousfish/modules/cryptography/EncryptorAdapter.java

```diff
@@ -1,17 +1,45 @@
 package com.nervousfish.nervousfish.modules.cryptography;
 
+import android.util.Base64;
+
+import com.nervousfish.nervousfish.exceptions.EncryptionException;
 import com.nervousfish.nervousfish.service_locator.IServiceLocator;
 import com.nervousfish.nervousfish.service_locator.ModuleWrapper;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.util.Random;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.SecretKey;
+import javax.crypto.SecretKeyFactory;
+import javax.crypto.spec.PBEKeySpec;
+import javax.crypto.spec.PBEParameterSpec;
+
 /**
  * An adapter to the default Java class for encrypting messages
  */
 public final class EncryptorAdapter implements IEncryptor {
     private static final long serialVersionUID = 5930930748980177440L;
     private static final Logger LOGGER = LoggerFactory.getLogger("EncryptorAdapter");
+    private static final String PBE_WITH_MD5_AND_DES = "PBEWithMD5AndDES";
+    private static final String UTF_8_NO_LONGER_SUPPORTED = "UTF-8 is no longer an encoding algorithm";
+    private static final String CANNOT_HAPPEN_UTF_8 = "Cannot happen, no encodign algorithm like UTF-8";
```

Comment:
I might be mistaken, but these all seem like messages you use for logging? I would suggest making them less generic and putting them back into their respective log statement